### ISSUE TITLE
STX address cut off when sending STX with connect on preview screen

### ISSUE THREAD

**Issue by timstackblock** [leather-io/extension#719]
STX address cut off when sending STX with connect on preview screen

FF plugin

<img width="499" alt="Screen Shot 2020-12-23 at 9 39 51 AM" src="https://user-images.githubusercontent.com/49165468/103008336-f8b82180-4502-11eb-8eda-85401106abe6.png">

**Comment by markmhendrickson** [comment]
@hstove is this ready for retesting?

**Comment by hstove** [comment]
Use #731 for QA

**Comment by timstackblock** [comment]
Cant test this because the send function is blocked by this issue

<img width="479" alt="Screen Shot 2021-01-04 at 9 09 06 AM" src="https://user-images.githubusercontent.com/49165468/103544803-a2d26a80-4e6e-11eb-94c0-62edafe5be18.png">

**Comment by timstackblock** [comment]
This looks like a network issue

**Comment by markmhendrickson** [comment]
The fix here appears merged into https://github.com/blockstack/ux/pull/725 so can get tested with that PR's build.
